app/templates/content/index.hbs

```diff
@@ -97,12 +97,18 @@
                         </div>
                     {{/if}}
 
-                    {{#if model.doi}}
-                        <div class="p-t-xs">
-                            <h4 class="p-v-md f-w-md"><strong>{{t "content.article_doi"}}</strong></h4>
+                    <div class="p-t-xs">
+                        {{#if model.doi}}
```

Comment:
model.doi is actually the user-provided DOI for a peer-reviewed publication (not the preprint).

Reply:
It might help to use `model.articleDoiUrl` here instead